tests/test_integration.py

```diff
@@ -228,15 +248,15 @@ def test_noise_model_qasm_simulator(self, monkeypatch):
             m.setattr(
                 qiskit_aer.AerSimulator, "set_options", lambda *args, **kwargs: cache.append(kwargs)
             )
-            dev = qml.device("qiskit.aer", wires=2, noise_model="test value")
+            qml.device("qiskit.aer", wires=2, noise_model="test value")
         assert cache[-1] == {"noise_model": "test value"}
 
     def test_invalid_noise_model(self):
         """Test that the noise model argument causes an exception to be raised
         if the backend does not support it"""
         dev_name = pldevices[1][0]
         with pytest.raises(AttributeError, match="field noise_model is not valid for this backend"):
-            dev = qml.device(dev_name, wires=2, noise_model="test value")
+            qml.device(dev_name, wires=2, noise_model="test value")
```

Comment:
No need for "dev"?

Reply:
Not really, for the same reason as above.